### ISSUE TITLE
PRMC_POSSIBLY_REDUNDANT_METHOD_CALLS false positive

### ISSUE THREAD

**Issue by efenderbosch** [spotbugs/spotbugs#841]
```java
return dbMapper.batchLoad(contactsIdsOnly).values().stream().
    flatMap(Collection::stream).
    filter(Contact.class::isInstance).
    map(Contact.class::cast). // PRMC_POSSIBLY_REDUNDANT_METHOD_CALLS here
    collect(toList());
```

javac 10.0.1
spotbugs 3.1.10
fb-contrib 7.4.3.sb
MacOS, doesn't seem to happen in our Linux CI builds

**Comment by KengoTODA** [comment]
This detector is provided by https://github.com/mebigfatguy/fb-contrib so I recommend you to issue at that project.
